.github/workflows/linux-nightly.yml

```diff
@@ -54,3 +55,49 @@ jobs:
           for pkg in fixed_dist/torcharrow*.whl; do
               ~/.local/bin/aws s3 cp "$pkg" "$S3_PATH" --acl public-read
           done
+
+  macos-container:
+    runs-on: macos-11
```

Comment:
shall we just use macos-latest?

Reply:
i tried that too and it defaults to macos-11 , also atm macos 12 breaks velox (we are fixing it) , so I dint want to set it to latest just yet.